website/docs/backends/types/index.html.md

```diff
@@ -1,25 +0,0 @@
----
-layout: "language"
-page_title: "Backend: Supported Backend Types"
-sidebar_current: "docs-backends-types-index"
-description: |-
-  Terraform can store the state remotely, making it easier to version and work with in a team.
----
-
-# Backend Types
```

Comment:
We may need to keep this explanation of backend types because I think standard and enhanced are discussed in the certification exam. Maybe the content can go on `/docs/language/settings/backends/index.html`? If it's already there I'm not finding it.